assets/src/dashboard/app/views/editorSettings/index.js

```diff
@@ -81,6 +84,8 @@ function EditorSettings() {
     maxUploadFormatted,
   } = useConfig();
 
+  const [activeDialog, setActiveDialog] = useState('');
```

Comment:
Could this be `showDialgo` and `setShowDialog` and just toggle from `true` to `false` instead of using strings and string compares?

Reply:
And just to clarify, I can see the active dialog being a string if there are multiple (or in the future could be multiple) dialog to pop open on this page, and therefore the name of the dialog to open is used to trigger the appropriate one.  If that was your intent (to future proof it) then I can dig it.

Reply:
Great call out and I'm sorry for not making this clearer in my description! My intention here is to future proof it (did the same for myStories/content). I think this view will get more dialogs as functionality increases.

Reply:
Using an ENUM to show dialogs is fine, but perhaps the default value here should be `null`.